lib/web/sessions.go

```diff
@@ -1042,9 +1042,6 @@ func (s *sessionCache) invalidateSession(ctx context.Context, sctx *SessionConte
 		return trace.Wrap(sessionDeletionErrs)
 	}
 
-	if err := s.releaseResources(sctx.GetUser(), sctx.GetSessionID()); err != nil {
-		return trace.Wrap(err)
-	}
```

Comment:
i thought this was okay. the function where this call resides, is only used in one area